WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -448,6 +450,9 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
 
     } else if (sec == PostSettingsSectionStickyPost) {
         return 1;
+
+    } else if (sec == PostSettingsSectionAuthor && self.apost.author) {
```

Comment:
Just realized there's a consequence of checking for an author here. On new Pages / Posts one isn't set. If this is removed the user can successfully set an author on creation, but the UI needs a little more polish. It'll probably be best to set the post's author beforehand.